app/views/groups/_groups_table_header.html.erb

```diff
@@ -0,0 +1,8 @@
+<tr>
+  <th><%= Group.human_attribute_name(:name)%></th>
+  <th><%= Group.human_attribute_name(:responsable_name)%></th>
+  <th><%= Group.human_attribute_name(:responsable_contact)%></th>
+  <th><%= Group.human_attribute_name(:from_date)%></th>
+  <th><%= Group.human_attribute_name(:to_date)%></th>
+  <th><%= Group.human_attribute_name(:notes)%></th>
+</tr>
```

Comment:
Is this partial used elsewhere besides the index view?
If not, a suggestion to just include this markup in the index view...

Reply:
Actually I don't think I will use again. I removed the partial and moved the header in the groups/index.html.erb.

Done with fixup in `405994f`